portfolio/src/main/webapp/map.js

```diff
@@ -1,58 +1,76 @@
-class MapHandler {
-  // The coordinates for all markers and their related properties.
-  static MAP_CENTER = {
-    lat: 43,
-    lng: 169,
-  };
-
-  static SUZHOU_COORD = {
-    lat: 31.293463,
-    lng: 120.666522,
-    title: 'My Hometown',
-    id: 'suzhou',
-  };
+/** 
+ *  The coordinate class which registers information for 
+ *  each marker. 
+ */
+class Coordinate {
+  
+  constructor(latitude, longitude, title, id) {
+    /** @member {float} latitude */
+    this.latitude = latitude;
+    /** @member {float} longitude */
+    this.longitude = longitude;
+    /** @member {String} title the note on the marker*/
+    this.title = title;
+    /** @member {id} id the div id associated with this Coordinate*/
+    this.id = id;
+  }
+}
 
-  static SMITH_COORD = {
-    lat: 42.319266,
-    lng: -72.642447,
-    title: 'Smith College',
-    id: 'smith',
-  };
+class MapHandler {
 
-  static RICE_COORD = {
-    lat: 29.717296,
-    lng: -95.399418,
-    title: 'Rice University',
-    id: 'rice',
-  };
+  // The coordinates for map center.
+  static MAP_CENTER = new Coordinate(43, 169, '', '');
+  // The coordinates for Suzhou, related div id and marker title.
+  static SUZHOU_COORD = 
+      new Coordinate(31.293463, 120.666522, 'My Hometown', 'suzhou');
+  // The coordinates for Smith College, related div id and marker title.
+  static SMITH_COORD = 
+      new Coordinate(42.319266, -72.642447, 'Smith College', 'smith');
+  // The coordinates for Rice University, related div id and marker title.
+  static RICE_COORD = 
+      new Coordinate(29.717296, -95.399418, 'Rice University', 'rice');
 
   constructor(document) {
-    this.previousSection = undefined;
-    this.document = document;
+    /** @member {String} previousSection_ the id of the div on display */
```

Comment:
Mark private members with @private annotations and if the values don't get modified, with @const annotations.

Reply:
done